packages/serverless-contracts/src/contracts/apiGateway/features/httpLambdaHandler.ts

```diff
@@ -0,0 +1,81 @@
+import Ajv from 'ajv';
+import createHttpError, { isHttpError } from 'http-errors';
+
+import { ApiGatewayContract } from '../apiGatewayContract';
+import {
+  BodyType,
+  CompleteHandlerType,
+  HandlerEventType,
+  HandlerType,
+  LambdaEventType,
+  LambdaReturnType,
+  OutputType,
+} from '../types';
+
+const proxyEventToHandlerEvent = <Contract extends ApiGatewayContract>({
+  requestContext,
+  body: proxyEventBody = null,
+  headers,
+  pathParameters,
+  queryStringParameters,
+}: LambdaEventType<Contract>): HandlerEventType<Contract> => {
+  return {
+    requestContext,
+    body: (proxyEventBody !== null
+      ? JSON.parse(proxyEventBody)
+      : undefined) as BodyType<Contract>,
+    headers,
+    pathParameters,
+    queryStringParameters,
+  } as unknown as HandlerEventType<Contract>;
+};
+
+const handlerResponseToLambdaResult = <Contract extends ApiGatewayContract>(
+  handlerResponse: OutputType<Contract>,
+): LambdaReturnType<Contract> => ({
+  statusCode: 200,
+  body: JSON.stringify(handlerResponse),
+});
+
+const ajv = new Ajv();
+
+export const getHttpLambdaHandler =
+  <Contract extends ApiGatewayContract>(contract: Contract) =>
+  (
+    handler: HandlerType<typeof contract>,
+  ): CompleteHandlerType<typeof contract> =>
+  async event => {
+    try {
+      const parsedEvent = proxyEventToHandlerEvent<Contract>(event);
+
+      if (contract.bodySchema !== undefined) {
```

Comment:
Maybe it can be a next feature, but for the moment, i wanted to implement what already exists with the middlewares we already use.
But indeed, we have the ability to validate other things than the body on input data